src/v2/mocks/iam-http.js

```diff
@@ -15,12 +15,12 @@ export default function createMockHttp() {
     namespaces: {
       kind: 'ProjectList',
       apiVersion: 'project.openshift.io/v1',
-      metadata: { selfLink: '/apis/project.openshift.io/v1/projects' },
+      metadata: { selfLink: 'selfLink has been removed in K8s 1.20' },
```

Comment:
Remove from mocks.  This should match what we get from kubernetes, which is no selfLink.